services/movie.py

```diff
@@ -0,0 +1,37 @@
+from db.models import Movie
+
+
+def get_movies(genres_ids: list[int] = None,
```

Comment:
It accepts Optional[list[int]] because None is default. Fix in all places(with all types)